packages/twenty-front/src/modules/ui/input/components/TextInputV2.tsx

```diff
@@ -191,7 +215,7 @@ const StyledTrailingIcon = styled.div<{
 
 const INPUT_TYPE_PASSWORD = 'password';
 
-export type TextInputV2Size = 'sm' | 'md' | 'lg';
```

Comment:
to double check with Thomas!

Reply:
Confirmed by @Bonapara  :

- 20px was used by `TextInputV2`
- 24px was not used before in `TextInputV2` but by `TextInput`
- 28px was used by `TextInputV2`
- 32px was used by `TextInputV2`

So it is legit to create the missing 24px size for our componnent by `TextInputV2` and to use it in our `OptionMenuV2`